eng/common/scripts/Invoke-GitHubAPI.ps1

```diff
@@ -65,7 +67,9 @@ function Invoke-GitHubAPIPatch {
       return $resp
     }
     else {
-      throw "Did not fire request because of empty body."
+      $warning = "{0} with Uri [ $apiURI ] did not fire request because of empty body." -f (Get-PSCallStack)[1].FunctionName
```

Comment:
Is it ever an expected case that the body is empty?

Reply:
no.

Reply:
Call to some of the function may contain empty parameters. I thought it was easier to exit quietly here other wish we will need many condition for the calls.

Reply:
If someone calls it with an empty body expecting it to do something that seems like an error case and we should probably still error. Although doing a throw here isn't that helpful inside of the try/catch block you have. You would need to move it outside of the try/catch block.

Reply:
So for example in the[ Submit-PullRequest.ps1](https://github.com/Azure/azure-sdk-tools/blob/c9ea0fb5f19d07ff4333e8ba9c4c171883f8dd26/eng/common/scripts/Submit-PullRequest.ps1#L90-L93) there are times when the script is called without `Labels` , `Assignees`, `Users` or  `Teams`. If we don't have this then we,'' have to condition each of those calls. I thought it was better to allow this case and just log that nothing happened because of no parameters set..